codex-rs/README.md

```diff
@@ -71,9 +71,13 @@ To test to see what happens when a command is run under the sandbox provided by
 
 ```
 # macOS
-codex debug seatbelt [--full-auto] [COMMAND]...
+codex sandbox macos [--full-auto] [COMMAND]...
```

Comment:
Since we're going to drop `--full-auto` soon, can you please just drop all this from the docs?